services/minespace-web/src/styles/overrides/antd-overrides.scss

```diff
@@ -283,7 +287,6 @@ h5.ant-typography,
   color: #494949;
   border-bottom: 2px solid $primary-color;
   border-radius: 0;
-  font-weight: 600;
```

Comment:
snuck in making the table headers not bold so that they take up less vertical space.